radicle-cli/src/commands/patch/list.rs

```diff
@@ -85,7 +85,7 @@ fn print(
 ) -> anyhow::Result<()> {
     let target_head = common::patch_merge_target_oid(patch.target(), storage)?;
 
-    let you = patch.author().id() == whoami;
+    let you = &**patch.author().id() == whoami;
```

Comment:
suggestion: have a method on `Did` for checking if a `PublicKey` is owned by that `Did`

Reply:
Hmmmm, I think the "ownership" relation might be a bit weird and come out of nowhere. I'd rather have maybe a `as_key()` method.

Reply:
But that's essentially what DIDs boil down to in a lot of cases, key ownership (and then associating actions across uses of those keys). But `as_key` also works, just goes in the opposite direction :)

Reply:
Yeah, it's just that right now, our `Did` type is basically `did:key`, so I find it a bit confusing to use the verb "owns" when they're kind of the same thing..